apps/meteor/client/views/admin/manageSubscription/ManageSubscriptionRoute.tsx

```diff
@@ -0,0 +1,18 @@
+import { usePermission } from '@rocket.chat/ui-contexts';
+import type { ReactElement } from 'react';
+import React, { memo } from 'react';
+
+import NotAuthorizedPage from '../../notAuthorized/NotAuthorizedPage';
+import ManageSubscriptionPage from './ManageSubscriptionPage';
+
+const ManageSubscriptionRoute = (): ReactElement => {
+	const canViewManageSubscription = usePermission('manage-cloud');
+
+	if (canViewManageSubscription) {
```

Comment:
I would suggest to replace the condition to render `<NotAuthorizedPage />` if there's no permission, and returning the `ManageSubscriptionPage` by default